ProjectSidewalk/SidewalkWebpage
# 3048 - fixed text for user dashboards based on labeling history

Resolves #3048 

Made two additional strings in dashboard.json to account for the three states a user's label history can be in.

1. No labelling mistakes/no labels
2. Has labelling mistakes but not of all types of labels
3. Has labelling mistakes of all types of labels.

Added some conditionals within MistakeCarousel.js to help determine which appropriate text should appear under "Your recent labeling mistakes"

##### Before/After screenshots (if applicable)

Before:

1. User had no mistakes labeling, still looked weird.
<img width="1187" alt="3048_original_no_mistakes_case" src="https://user-images.githubusercontent.com/56641275/222253501-e57a0500-5177-4921-993b-74870064803f.png">

2. User has made a mistake labeling all types of labels, also looks weird.
![3048_weird_edge_case](https://user-images.githubusercontent.com/56641275/222253647-a78e13d2-d1a2-4359-921b-6eadcd1dde81.png)

After:

1. This resolves the user has no mistakes labeling case
<img width="695" alt="3048_fix_no_mistakes" src="https://user-images.githubusercontent.com/56641275/222253919-b19a52ea-420a-4d0d-9337-d28ad6023c26.png">
2. This resolves the case where the user has made a mistake for all types of labels 
<img width="1280" alt="3048_fix_all_mistakes_case" src="https://user-images.githubusercontent.com/56641275/222253971-3d6b42e7-4839-4cb4-bb71-d6c3d6c81653.png">
3. Standard text from prior that still functions the same regarding labels that have no mistakes still.
<img width="1257" alt="3048_1_mistake_case" src="https://user-images.githubusercontent.com/56641275/222254008-5ef640bb-657c-4293-a671-60f80cc543ce.png">
<img width="1240" alt="3048_2_mistake_case" src="https://user-images.githubusercontent.com/56641275/222254030-641e5571-59b0-49f7-98c5-bfc36908d4d5.png">

##### Testing instructions
1. Sign into ProjectSidewalk and go to dashboard
2. Confirm accurate text regarding user labeling mistakes.

##### Things to check before submitting the PR
